python/dolfinx/mesh.py

```diff
@@ -439,7 +440,18 @@ def create_mesh(
     return Mesh(mesh, domain)
 
 
-def create_submesh(msh, dim, entities):
+def create_submesh(msh: Mesh, dim: int, entities: npt.NDArray[np.int32]):
+    """Create a mesh based on a subset of entities from an existing mesh.
```

Comment:
```suggestion
    """Create a mesh that with specified entities from another mesh.
```

Reply:
To me that new sentence doesn’t make sense.